spec/javascripts/controllers/ProductsController_spec.js

```diff
@@ -0,0 +1,33 @@
+describe("ProductsController", function() {
+  beforeEach(module("pinterest"));
+
+  var $controller, $httpBackend, products, $resource, $scope;
+
+  products = [
+    { id: 1, blurb: "blurb 1", details_url: "", title: "Prod 1", thumbnail_url: "" },
+    { id: 2, blurb: "blurb 2", details_url: "", title: "Prod 2", thumbnail_url: "" },
```

Comment:
Line is too long.<br>Identifier 'details_url' is not in camel case.<br>Identifier 'thumbnail_url' is not in camel case.